pkg/util/docker/containers.go

```diff
@@ -171,12 +171,16 @@ func (d *DockerUtil) dockerContainers(cfg *ContainerListConfig) ([]*containers.C
 
 // Parse the health out of a container status. The format is either:
 //  - 'Up 5 seconds (health: starting)'
+//  - 'Up 18 hours (unhealthy)'
 //  - 'Up about an hour'
 func parseContainerHealth(status string) string {
 	// Avoid allocations in most cases by just checking for '('
 	if strings.IndexByte(status, '(') == -1 {
 		return ""
 	}
+	if strings.Index(status, "unhealthy") >= 0 {
```

Comment:
I think you should put this check before `if strings.IndexByte(status, '(') == -1 {`

And also it may be good to add a unit test to validate your logic.